examples/advanced-project-js/playwright.config.js

```diff
@@ -0,0 +1,21 @@
+const { defineConfig, devices } = require('@playwright/test');
+
+const config = defineConfig({
+  timeout: 30000,
+  use: {
+    baseURL: 'https://www.danube-web.shop',
+    viewport: { width: 1280, height: 720 },
+  },
+  projects: [
+    {
+      testDir: './src/playwright/',
+      name: 'Playwright tests',
+      use: {
+        ...devices['Desktop Chrome'],
+        userAgent: `${devices['Desktop Chrome'].userAgent} (Checkly, https://www.checklyhq.com)`
```

Comment:
@maxigimenez @MariadeAnton Is this something we recommend people to do? If we ship this in the general boilerplate we should highlight in the docs, too.

Reply:
@stefanjudis https://www.checklyhq.com/docs/platform/allowlisting-traffic/#allowlisting-playwright-checks-with-user-agents, we are adjusting the docs still btw